docs/rest_api/contacts.rst

```diff
@@ -246,6 +246,8 @@ Get a list of Contacts.
      - An array of advanced where conditions
    * - ``order``
      - An array of advanced order statements
+   * - ``select``
+     - An array of field aliases to include in the lead data, ID is required. If not provided, all fields are returned.
```

Comment:
```suggestion
     - An array of field aliases to include in the Contact data, with ID a mandatory field. Mautic returns all fields if it's not provided.
```

I think this should be Contact, or are we referring here to some legacy code where it still refers to lead? Also fixed the Vale warning about passive voice.

Reply:
Thank you for your response, it has now been updated.